### PR TITLE
Web log: allow filtering with regex

### PR DESCRIPTION
**Pull request by l2isbad**
https://github.com/firehol/netdata/issues/2278
https://github.com/firehol/netdata/issues/2192
> An easy way to do it, is to add a filter regex configuration option. So, we could add the same log file many times, with different filters.

^

### CODE REVIEW

File: conf.d/python.d/web_log.conf [hunk 1]
```diff
@@ -60,9 +60,12 @@
...
 #
 #     path: 'PATH'                        # the path to web server log file
-#     detailed_response_codes: yes/no     # Default: yes. Additional chart where response codes are not grouped
-#     detailed_response_aggregate: yes/no # Default: yes. Not aggregated detailed response codes charts
-#     all_time : yes/no                   # Default: yes. All time unique client IPs chart (50000 addresses ~ 400KB)
+#     detailed_response_codes: yes/no     # default: yes. Additional chart where response codes are not grouped
+#     detailed_response_aggregate: yes/no # default: yes. Not aggregated detailed response codes charts
+#     all_time : yes/no                   # default: yes. All time unique client IPs chart (50000 addresses ~ 400KB)
+#     filter:                             # filter with regex
+#          inclue: 'REGEX'                # only those rows that matches the regex
```
Comment (ktsaou) [inline conf.d/python.d/web_log.conf]: a typo

**Comment by ktsaou** [comment]
@l2isbad codacy says: Redefining built-in 'filter'
Is this ok?

**Comment by l2isbad** [comment]
@ktsaou ready

**Comment by ktsaou** [comment]
you are the best!

**Pull request merged**